images/icon-dark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M14 3l-3 3V4l-1-1-9 9 1 1h12l1-1V4zm-4 9H6.4L10 8.4zm0-5l-5 5H2.4L10 4.4zm4 5h-3V7.4l3-3z" fill="#c5c5c5"/></svg>
```

Comment:
A mentioned concern might resolve by using our brand color `#0288d1` instead of monochrome stroke.